sympy/functions/elementary/trigonometric.py

```diff
@@ -375,7 +375,7 @@ def _eval_rewrite_as_Pow(self, arg):
             return I*x**-I / 2 - I*x**I /2
 
     def _eval_rewrite_as_cos(self, arg):
-        return -cos(arg + S.Pi/2)
+        return -cos(arg + S.Pi/2, evaluate=False)
```

Comment:
Okay then it must be changed to `cos(arg - S.Pi/2, evaluate=False)`

```
In [12]: cos(x - S.Pi/2, evaluate=False)
Out[12]: 
   ⎛    π⎞
cos⎜x - ─⎟
   ⎝    2⎠

```

I will do this in next commit. Thanks!